### PR TITLE
Adding news for 0.14.0

### PR DESCRIPTION
**Pull request by sishida**
Signed-off-by: Sarah Ishida <sishida@us.ibm.com>

PR for issue eclipse/codewind#3159

Adding news page for 0.14.0.

### CODE REVIEW

File: docs/_news/news14.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+##### Eclipse, IntelliJ, and VS Code
+- With [secure template sources](https://github.com/eclipse/codewind/issues/2647), add template sources that require authentication.
+  - When you add a new template source repository, provide authentication with a username and password or use an access token.
+  - You can also edit template sources to update credentials.
+
+##### Eclipse and VS Code
+- Link Codewind projects together with [project linking](https://github.com/eclipse/codewind/issues/2852).
+  - When you link a project to another project, the project that is the link target has the other project's hostname exposed in an environment variable that you define.
+  - For example, linking **Project A** to **Project B** through the `PROJECT_A` environment variable means that **Project B** can access the code of the `PROJECT_A` environment variable to get the current hostname of **Project B**. Then, **Project A** can use **Project B**.
```
Comment (eharris369) [inline docs/_news/news14.md:23]: This is a bit confusing. Did you mean that Project A can use the environment variable to get the current hostname of Project B?

File: docs/_news/news14.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  - Right-click your project and click **Manage Project Links**. Use **Add**, **Edit**, and **Remove** to manage links.
```
Comment (eharris369) [inline docs/_news/news14.md:24]: @tetchel Does this apply to VSCode as well?

Comment (tetchel) [reply docs/_news/news14.md:24]: Mostly. In VS Code it's called "Link Project" to create a new link. To "manage links" for a project, use the project overview page. Refer back to james's screenshots https://github.com/eclipse/codewind/issues/2852#issuecomment-649414121

File: docs/_news/news14.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- [Manage template sources in IntelliJ](https://github.com/eclipse/codewind/issues/985).
+
+##### VS Code
+- The [changelog](https://github.com/eclipse/codewind-vscode/wiki/Changelog#0140-july-2020) file is updated for Codewind 0.14.0. 
+- The Project Overview page has undergone [another visual update](https://github.com/eclipse/codewind-vscode/pull/620).
+- You can [right-click many fields](https://github.com/eclipse/codewind/issues/1153) in the Project Overview and Template Sources pages to copy the values.
+
+#### List of Fixes
+- In Codewind for VS Code:
+    - [After you manually edit the template sources in the Template Source Manager, you won't be prompted to select one in the Command Palette](https://github.com/eclipse/codewind/issues/3165).
```
Comment (tetchel) [inline docs/_news/news14.md:37]: https://github.com/eclipse/codewind/issues/3158 is also fixed in this release

**Pull request merged**